pkg/middlewares/tracing/wrapper.go

```diff
@@ -10,12 +10,12 @@ import (
 	"github.com/traefik/traefik/v2/pkg/tracing"
 )
 
-// Tracable embeds tracing information.
-type Tracable interface {
+// Traceable embeds tracing information.
+type Traceable interface {
```

Comment:
It's possible this is a public interface and thus can't be changed, it's trivial to drop such things.